Update examples description of "svg element with explicit role has accessible name" (7d6734) (#1217)
act-rules/act-rules.github.io: _rules/explicit-SVG-image-has-name-7d6734.md

## Applicability

The rule applies to any element in the [SVG](https://www.w3.org/2000/svg) namespace with an [explicit semantic role][explicit role] of either `img`, `graphics-document`, `graphics-symbol`, that is [included in the accessibility tree][].

> **Note:** The [SVG Accessibility API Mappings, Section 5.1.2 Including Elements in the Accessibility Tree](https://www.w3.org/TR/svg-aam-1.0/#include_elements) specifies that many elements in the SVG namespace are purely presentational and should not be included in the accessibility tree unless indicated otherwise through the use of text alternative content, an explicit WAI ARIA role, or a valid `tabindex` attribute.

Reviewer: We should describe the exceptions that could include the elements in the SVG namespace in the accessibility tree. What about this?
```suggestion
**Note:** The [SVG Accessibility API Mappings, Section 5.1.2 Including Elements in the Accessibility Tree](https://www.w3.org/TR/svg-aam-1.0/#include_elements) specifies that many elements in the SVG namespace are purely presentational and should normally omitted from the accessibility tree unless indicated otherwise through the use of text alternative content (one direct non-empty child `title` element or `desc` element), an explicit WAI ARIA role (a non-empty `aria-label` attribute or `aria-roledescription` attribute or an `aria-labelledby` attribute or `aria-describedby` attribute containing valid IDREF tokens), or a valid integer `tabindex` attribute.
```

Author: I disagree.
I don't think this rule is the place to list all the ways to give an accessible name or an explicit role to an element.

#### Failed Example 3

> This `circle` element has an [explicit role][] of `graphics-symbol` but does not have an [accessible name][].

Reviewer: To make it consistent with the description of failed example 1:
```suggestion
This `circle` element has an [explicit role][] of `graphics-symbol` but has no [accessible name][].
```

#### Inapplicable Example 1

> Neither the `svg` element nor the `circle` element has an [explicit role][].

Reviewer: Make it more clear and consistent with other example i.e. inapplicable example 3.
```suggestion
Neither the `svg` element nor the `circle` element has any of the [explicit role][] of  `img`, `graphics-document`, `graphics-symbol`.
```
